src/shared/components/search-filters/search-filters.component.html

```diff
@@ -25,6 +26,6 @@
     </div>
   </ng-container>
 
-  <button [disabled]="!isSearchableAndSearchInputsReady()" type="button" class="button" (click)="apply()" (keyup.enter)="apply()" >Apply</button> &nbsp;&nbsp;&nbsp;
-  <button id="reset" type="button" (click)="reset()" class="button button-secondary">Reset</button>
+  <button [disabled]="!isSearchableAndSearchInputsReady()" type="button" class="button" (click)="apply()" (keyup.enter)="apply()" title="Apply filter" aria-label="Apply filter" >Apply</button> &nbsp;&nbsp;&nbsp;
```

Comment:
_Being annoying:_
What's that extra space at the end of the tag for? Future-proofing? Breathing room? Is the `aria-label` claustrophobic? 🤔

Reply:
Removed. No idea where that came from (my bad)